src/xo-app/dashboard/health/index.js

```diff
@@ -445,11 +451,42 @@ export default class Health extends Component {
 
   _getSrUrl = sr => `srs/${sr.id}`
 
+  _getPoolPredicate = createSelector(
+    createSelector(() => this.state.pools, resolveIds),
+    poolIds =>
+      isEmpty(poolIds) ? undefined : item => includes(poolIds, item.$pool)
+  )
+
+  _getFilteredCollections = createSelector(
```

Comment:
What's the point of this wrapper selector?

Reply:
To avoid  building the collections array in the render.

Reply:
You are not using this array.

Reply:
It used in the line 479.

Reply:
You do not need this intermediary.